c_api/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT OR Apache-2.0"
 
 [lib]
 name = "c2pa_c"
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "staticlib"]
```

Comment:
@scouten-adobe @gpeacock Remind me why we had only `cdylib` (if there was a reason) ? Any concern changing that?

Reply:
I measured clean build times with the `staticlib` setting and without and there's no appreciable difference. I'm :+1: on this change.